docs/documentation/usage.md

```diff
@@ -106,3 +106,22 @@ which can be handy for generating results multiple times.
     ``` kotlin
     val faker = Faker(Random(0))
     ```
+
+## Random Java enum values
+
+To get a random value from a Java `enum` you can employ the `Options.option()` method.
+
+Assume the following enum:
+
+```java
+enum Day {
+    MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY
+}
+```
+
+To retrieve a random Day:
+
+```java
+private final Options opt = faker.options();
+opt.option(Day.class);
+```
```

Comment:
Does this need the `=== "Java"` heading to render properly on the docs site?

Reply:
i guess it is required for specific tab naming, especially when there are several code snippets 

may be @bodiam has more insights

Reply:
Okay I tweaked the text and example to now be within a `=== "Java"` block

Reply:
That is correct, but you should be able to run the docs locally right, but running `mkdocs serve` or something like that.

Reply:
I have no idea, I've never tried. I have a Linux VM I could try on I guess.

Reply:
It's not that important, and if the formatting is a bit off, we'll fix it, no problem! Thanks for this PR!

Reply:
Thanks! If you're good with the content, I'll go ahead and merge it?

Reply:
Yes :)

Reply:
I'll try to deploy this later, it's still something to automate.